api-proxy/src/reqhandler.mjs

```diff
@@ -0,0 +1,136 @@
+import * as fs from 'fs'
+const fsp = fs.promises
+import * as https from 'https'
+import * as stream from 'stream'
+import * as nu from 'util'
+import * as zlib from 'zlib'
+
+const omit = ks => obj =>
+  Object.keys(obj).reduce((acc, k) => ks.includes(k) ? acc : ({...acc, [k]: obj[k]}), {})
+
+const promiseEvent = eventName => x => new Promise(resolve => {
+  x.on(eventName, a => resolve(a))
+})
+
+const quoteSingles = x => x.replace(/[']/g, '\\\'')
+const quoteBackquotes = x => x.replace(/[`]/g, '\\`')
+const formatObj = x => nu.inspect(x, {depth: Infinity, maxArrayLength: 10e3, breakLength: 100})
```

Comment:
what does this do?

Reply:
`inspect` is a (awesome) Node utility that formats objects in a very readable way while still allowing them to be read via Node's interpreter. I've included a few options to ensure the output is less concise so it includes all of the data, but this is the workhorse that produces the fairly readable handlers directly from code.